src/applications/personalization/preferences/SetPreferences.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Link, withRouter } from 'react-router';
+import { connect } from 'react-redux';
+
+import { chunk, fill } from 'lodash';
+
+import { benefitChoices } from './helpers';
+
+import { setPreference, savePreferences, fetchPreferences } from './actions';
+
+import localStorage from '../../../platform/utilities/storage/localStorage';
+
+import Checkbox from '../../gi/components/Checkbox.jsx';
+
+class SetPreferences extends React.Component {
+  componentDidMount() {
+    this.props.fetchPreferences();
+    localStorage.setItem('dashboardLastVisitedAt', Date.now());
+  }
```

Comment:
`dashboardLastVisitedAt ` could be something to move into the `fetchPreferences ` action and just abstract the localStorage that much further.

Reply:
Good call, updated.